src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -1,4 +1,88 @@
 package com.kodilla.ecommercee.domain;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "USERS")
 public class User {
-}
+    private Long id;
+    private String username;
+    private String status;
+    private Long userKey;
+    private List<Order> orders = new ArrayList<>();
```

Comment:
celowo nie ustawiasz tej listy w konstruktorze ? W encji Cart  leci w konstruktorze lista... tutaj nie :) pytanie czy ta różnica jest uzasadniona? Jeśli robimy konstruktor bezparametrowy i konstruktor, który ustawia wszystkie parametry to raczej to powinno być.Tak musimy stworzyć entity przez konstruktor i listę zasilić osobno pod wywołaniem konstruktora.